ts/Core/Axis/ScrollbarAxis.ts

```diff
@@ -87,13 +87,15 @@ class ScrollbarAxis {
                             Math.min(
                                 axisMin,
                                 axis.min as any,
-                                axis.dataMin as any
+                                (axis.dataMin as any) > (axis.threshold as any) ?
+                                    axis.threshold : (axis.dataMin as any)
```

Comment:
Wouldn't it be more compact and readable to write `Math.min(axis.threshold, axis.dataMin)` in this expression and the three others?

Reply:
`Math.min` return `NaN` if any of the values are undefined.

Reply:
Maybe something like `Math.min(pick(axis.threshold, Number.MAX_VALUE), axis.dataMin)` could work though.

Reply:
Or not, tests are failing because `Math.min` turns `null` into `0` :/